diff --git a/components/eam/src/physics/cam/clubb_intr.F90 b/components/eam/src/physics/cam/clubb_intr.F90
index a93331fabd..80d34e7bd7 100644
--- a/components/eam/src/physics/cam/clubb_intr.F90
+++ b/components/eam/src/physics/cam/clubb_intr.F90
@@ -2969,6 +2969,7 @@ end subroutine clubb_init_cnst
            vmag_gust_dp(i) = ugust(min(prec_gust(i),6.94444e-4_r8)) ! Limit for the ZM gustiness equation set in Redelsperger et al. (2000)
            vmag_gust_dp(i) = max(0._r8, vmag_gust_dp(i) )
            vmag_gust_cl(i) = sqrt(max(0._r8,up2b(i)+vp2b(i)))
+           ! vmag_gust_cl(i) = min(6.53197_r8,sqrt(max(0._r8,up2b(i)+vp2b(i))))
            vmag_gust(i)    = sqrt(gust_facc * vmag_gust_cl(i)**2 &
                 + gust_fac(i) * vmag_gust_dp(i)**2)
           do k=1,pver
diff --git a/components/elm/src/biogeophys/CanopyFluxesMod.F90 b/components/elm/src/biogeophys/CanopyFluxesMod.F90
index 6c96e70e17..d5b3f21ef8 100755
--- a/components/elm/src/biogeophys/CanopyFluxesMod.F90
+++ b/components/elm/src/biogeophys/CanopyFluxesMod.F90
@@ -40,6 +40,7 @@ module CanopyFluxesMod
   use ColumnDataType        , only : col_es, col_ef, col_ws
   use VegetationType        , only : veg_pp
   use VegetationDataType    , only : veg_es, veg_ef, veg_ws, veg_wf
+  use VegetationDataType    , only : veg_cs
 
   !!! using elm_instMod messes with the compilation order
   use elm_instMod           , only : alm_fates, soil_water_retention_curve
@@ -322,6 +323,7 @@ contains
     !------------------------------------------------------------------------------
 
     associate(                                                               &
+         deadstemc            => veg_cs%deadstemc                             , & ! Input:  [real(r8) (:) ] (gC/m2) dead stem C
          snl                  => col_pp%snl                                   , & ! Input:  [integer  (:)   ]  number of snow layers
          dayl                 => grc_pp%dayl                                  , & ! Input:  [real(r8) (:)   ]  daylength (s)
          max_dayl             => grc_pp%max_dayl                              , & ! Input:  [real(r8) (:)   ]  maximum daylength for this grid cell (s)
@@ -463,7 +465,8 @@ contains
          p = filter_nolakeurbanp(fp)
          c = veg_pp%column(p)
          t = veg_pp%topounit(p)
-         if (frac_veg_nosno(p) == 0) then
+         ! if (frac_veg_nosno(p) == 0) then
+         if (frac_veg_nosno(p) == 0 .or. deadstemc(p) < 0._r8) then
             btran(p) = 0._r8
             t_veg(p) = forc_t(t)
             cf_bare  = forc_pbot(t)/(SHR_CONST_RGAS*0.001_r8*thm(p))*1.e06_r8
diff --git a/components/elm/src/biogeophys/CanopyTemperatureMod.F90 b/components/elm/src/biogeophys/CanopyTemperatureMod.F90
index 017c3770a9..e89accb945 100644
--- a/components/elm/src/biogeophys/CanopyTemperatureMod.F90
+++ b/components/elm/src/biogeophys/CanopyTemperatureMod.F90
@@ -29,6 +29,7 @@ module CanopyTemperatureMod
   use ColumnDataType       , only : col_es, col_ef, col_ws
   use VegetationType       , only : veg_pp
   use VegetationDataType   , only : veg_es, veg_ef, veg_wf
+  use VegetationDataType   , only : veg_cs
   !
   ! !PUBLIC TYPES:
   implicit none
@@ -112,6 +113,7 @@ contains
     !------------------------------------------------------------------------------
 
     associate(                                                          &
+         deadstemc        =>    veg_cs%deadstemc                      , & ! Input:  [real(r8) (:) ] (gC/m2) dead stem C
          snl              =>    col_pp%snl                            , & ! Input:  [integer  (:)   ] number of snow layers
          dz               =>    col_pp%dz                             , & ! Input:  [real(r8) (:,:) ] layer depth (m)
          zii              =>    col_pp%zii                            , & ! Output: [real(r8) (:)   ] convective boundary height [m]
@@ -402,6 +404,7 @@ contains
          if( .not.(veg_pp%is_fates(p))) then
             z0m(p)    = z0mr(veg_pp%itype(p)) * htop(p)
             displa(p) = displar(veg_pp%itype(p)) * htop(p)
+            if (deadstemc(p) < 0._r8) displa(p) = 0._r8
          end if
       end do
 
