From 8b3b50b804aba393896addcd90fa740ae338d70e Mon Sep 17 00:00:00 2001
From: chengzhuzhang <zhang40@llnl.gov>
Date: Sun, 19 Jul 2020 23:09:48 -0700
Subject: [PATCH] one more change on version before releasing

---
 acme_diags/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/acme_diags/__init__.py b/acme_diags/__init__.py
index db469b6d..ca4b7477 100644
--- a/acme_diags/__init__.py
+++ b/acme_diags/__init__.py
@@ -1,5 +1,5 @@
 import os
 import sys
 
-__version__ = 'v2.0.0'
+__version__ = 'v2.1.0'
 INSTALL_PATH = os.path.join(sys.prefix, 'share/e3sm_diags/')
