*** datm.buildnml.csh.orig	2010-06-11 10:59:29.246523532 -0600
--- datm.buildnml.csh	2010-06-11 11:06:30.710784206 -0600
***************
*** 34,48 ****
     streams        = 'clm_qian.T62.stream.Solar.txt 1895 1948 1972 ',
                      'clm_qian.T62.stream.Precip.txt 1895 1948 1972 ',
                      'clm_qian.T62.stream.TPQW.txt 1895 1948 1972 ',
!                     'presaero.stream.txt 1849 1849 2006'
     vectors        = 'null'
     mapmask        = 'nomask',
                      'nomask',
                      'nomask',
                      'nomask'
     tintalgo       = 'coszen',
                      'nearest',
                      'linear',
                      'linear'
    /
  EOF1
--- 34,56 ----
     streams        = 'clm_qian.T62.stream.Solar.txt 1895 1948 1972 ',
                      'clm_qian.T62.stream.Precip.txt 1895 1948 1972 ',
                      'clm_qian.T62.stream.TPQW.txt 1895 1948 1972 ',
!                     'presaero.stream.txt 1849 1849 2006',
!                     'datm.global1val.stream.CO2.txt 1766 1766 2005 '
     vectors        = 'null'
     mapmask        = 'nomask',
                      'nomask',
                      'nomask',
+                     'nomask',
                      'nomask'
+    mapalgo        = 'bilinear',
+                     'bilinear',
+                     'bilinear',
+                     'bilinear',
+                     'nn'
     tintalgo       = 'coszen',
                      'nearest',
                      'linear',
+                     'linear',
                      'linear'
    /
  EOF1
***************
*** 1112,1121 ****
--- 1120,1132 ----
  </streamstemplate>
  EOF1
  
+ cp $CASEBUILD/co2_streams.txt datm.global1val.stream.CO2.txt
+ 
  
  $CASETOOLS/listfilesin_streams -input_data_list -t clm_qian.T62.stream.Solar.txt >> $CASEBUILD/datm.input_data_list
  $CASETOOLS/listfilesin_streams -input_data_list -t clm_qian.T62.stream.Precip.txt >> $CASEBUILD/datm.input_data_list
  $CASETOOLS/listfilesin_streams -input_data_list -t clm_qian.T62.stream.TPQW.txt >> $CASEBUILD/datm.input_data_list
+ $CASETOOLS/listfilesin_streams -input_data_list -t datm.global1val.stream.CO2.txt  >> $CASEBUILD/datm.input_data_list
  
  cat >! presaero.stream.txt << EOF1
  <streamstemplate>
